crates/fbuild-build/src/lite_scons_harness.py

```
@@ -0,0 +1,737 @@
+    script = sys.modules.get("SCons.Script") or types.ModuleType("SCons.Script")
+
+    def Import(*names):
+        import inspect
+        frame = inspect.currentframe().f_back
+        if frame is None:
+            return
+        for nm in names:
+            if nm in ("env", "projenv", "DefaultEnvironment"):
+                frame.f_globals[nm] = env
+
+    setattr(script, "Import", Import)
```

Ruff (0.15.15)

[warning] Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)